SG14_test/slot_map_test.cpp

```diff
@@ -446,6 +446,31 @@ void BoundsCheckingTest()
     assert(cit == sm.end());
 }
 
+template<class SM>
+void GenerationTest()
+{
+    using T = typename SM::mapped_type;
+    constexpr size_t num_iter = 10;
```

Comment:
What you propose doesn't test the bug fix, though it might be a good test on its own. I've updated the test to do a little more, but there isn't much to test. Just make sure the generations are sequential and aren't skipping values.